.pre-commit-config.yaml

```diff
@@ -0,0 +1,64 @@
+repos:
+-   repo: https://github.com/pre-commit/pre-commit-hooks
+    rev: v4.4.0
+    hooks:
+    # General
+    -   id: check-merge-conflict
+        fail_fast: true
+    -   id: check-case-conflict
+        fail_fast: true
+    -   id: check-added-large-files
+    -   id: check-symlinks
+    -   id: destroyed-symlinks
+    # Syntax
+    -   id: check-json
```

Comment:
Do we have json files?

Reply:
And while we're here, do we want to ad `check-toml` too?

Reply:
It just passes without doing anything if not. I more added that in as a just in case. We could add `check-toml`; I overlooked it because I have something else checking `pyproject.toml` specifically

Reply:
Before I do add `check-toml`; are there any other checks you think we should add? A decent list is here: https://pre-commit.com/hooks.html (It has search functionality too and per-language searching too).

We could add some like [github.com/dannysepler/rm_unneeded_f_str](https://github.com/dannysepler/rm_unneeded_f_str)
> Find and replace unneeded f-strings in your code.